### PR TITLE
etcd: add periodic tests ci-etcd-unit-test-amd64 and ci-etcd-robustness-amd64

### PR DESCRIPTION
**Pull request by siyuanfoundation**
https://github.com/kubernetes/k8s.io/issues/6102

### CODE REVIEW

File: config/jobs/etcd/etcd-periodics.yaml [hunk 1]
```diff
@@ -30,3 +30,71 @@ periodics:
...
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240223-1ded72f317-master
+      command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        apt update && apt-get --yes install cmake libfuse3-dev libfuse3-3 fuse3
+        sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf
```
Comment (jmhbnz) [inline config/jobs/etcd/etcd-periodics.yaml:84]: Have we tested this with a local prow instance? I had a lot of trouble getting lazyfs to work within github actions. Curious to see how it behaves here.

**Review by jmhbnz — COMMENTED** [review]
/lgtm 

Thanks @siyuanfoundation - Question re lazyfs below but overall this looks good.

File: config/jobs/etcd/etcd-periodics.yaml (hunk 1 above)
Comment (siyuanfoundation) [reply config/jobs/etcd/etcd-periodics.yaml:84]: Yes, I have tested this locally. Had some trouble getting lazyfs to work too, but it seems to work after setting `privileged: true`

**Pull request merged**